src/main/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/config/DomainEventUrlConfig.kt

```diff
@@ -0,0 +1,45 @@
+package uk.gov.justice.digital.hmpps.approvedpremisesapi.config
+
+import org.springframework.boot.context.properties.ConfigurationProperties
+import org.springframework.context.annotation.Configuration
+import uk.gov.justice.digital.hmpps.approvedpremisesapi.jpa.entity.DomainEventType
+
+@Configuration
+@ConfigurationProperties(prefix = "url-templates.api")
+class DomainEventUrlConfig {
+  lateinit var cas1: Map<String, String>
```

Comment:
this may be a headache but it would be really nice to use UrlTemplate instead of String for the map value? I think spring should be able to do the mapping automatically